sys-devel/gdb/gdb-11.2.ebuild

```
@@ -154,6 +154,13 @@ src_configure() {
 		# Ideally we would like automagic-or-disabled here.
 		# But the check does not quite work on i686: bug #760926.
 		$(use_enable cet)
+
+		# We need to set both configure options, --with-sysroot and --libdir,
+		# to fix cross build issues that happen when configuring gmp.
+		# For some reason, it is not enough to set only --with-sysroot,
+		# also not enough to pass --with-gmp-xxx options.
+		--with-sysroot="${ESYSROOT}"
+		--libdir=${ESYSROOT}/usr/lib64
```

libdir isn't always `lib64`:
```suggestion
		--libdir="${ESYSROOT}/usr/$(get_libdir)"
```

---

Oh, actually, I think this might ought to be EPREFIX, as I think it's about where we install files. Or does that not work?

---

> libdir isn't always lib64:

Oh, you are right. Fixed it to `$(get_libdir)`. Thanks.

> Oh, actually, I think this might ought to be EPREFIX, as I think it's about where we install files. Or does that not work?

Already tried doing with `EPREFIX`, but that does not work at all.

`EROOT` works as well though.
I chose `ESYSROOT`, which looked like a better way with EAPI=7.

---

Yeah, if we really need it, ESYSROOT > EROOT (which is not well-defined in src_*). Could you put a comment above it just explaining briefly that we really do need --libdir there too?

(Just to emphasise it is not a mistake and the problem.0

Thanks!